tests/FSharp.Compiler.ComponentTests/Language/ComputationExpressionTests.fs

```diff
@@ -214,6 +214,70 @@ let run r2 r3 =
             (Error 3345, Line 18, Col 9, Line 18, Col 13, "use! may not be combined with and!")
         ]
         
```

Comment:
Please do add positive tests which include a `run` operation and demonstrate that a resource which is being `use! _ =`'d get's cleaned up with a custom `.Using` method of the builder.